Navigation/Navigation/Profile/Realm/Service.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  Service.swift
+//  Navigation
+//
+//  Created by Sokolov on 19.01.2023.
+//
+
+import Foundation
+import RealmSwift
+
+class Service {
+    
+    let realm = try! Realm()
```

Comment:
Non-frozen `RLMRealm` instances are thread-confined and cannot be
 shared across threads or dispatch queues. Trying to do so will cause an
 exception to be thrown. You must obtain an instance of `RLMRealm` on each
 thread or queue you want to interact with the Realm on. Realms can be confined
 to a dispatch queue rather than the thread they are opened on by explicitly
 passing in the queue when obtaining the `RLMRealm` instance. If this is not
 done, trying to use the same instance in multiple blocks dispatch to the same
 queue may fail as queues are not always run on the same thread.